README.md

```diff
@@ -104,3 +104,23 @@ vagrant up stage
 ```
 vagrant destroy --force stage
 ```
+
+## Monitoring
+
+* Prometheus
+```
+# Dashboard
+localhost:9090
```

Comment:
this is true for dev environment. what about stage/prod?
what about verifying the ports are open in stage/prod?

Reply:
Updated the Readme for @Balex3000 Doc.
Also tested in stage ENV by Alex.